app/src/main/java/org/oppia/app/shim/ViewComponentFactory.kt

```diff
@@ -3,6 +3,8 @@ package org.oppia.app.shim
 import android.view.View
 import org.oppia.app.view.ViewComponent
 
+/** Removes [InjectableFragment] dependency from Views */
+// TODO(#1619): Remove file post-Gradle
```

Comment:
Eh, let's keep it actually. Abstracting away the component factory like this can be handy.

Reply:
Sounds good